fe/fe-core/src/main/java/org/apache/doris/analysis/VirtualSlotRef.java

```diff
@@ -122,4 +105,9 @@ public void analyzeImpl(Analyzer analyzer) throws AnalysisException {
     public String getExprName() {
         return super.getExprName();
     }
+
+    @Override
+    public boolean supportSerializable() {
+        return false;
```

Comment:
Why return false?
It has a `write()` method before

Reply:
this subclass should not support Serializable, because it is generated in analyze.